CONTRIBUTING.md

```diff
@@ -0,0 +1,124 @@
+Contributing to OKD4
+====
+
+# Introduction into release payloads
+
+Unlike other Kubernetes distributions OKD4 has a strictly defined set of software to be installed.
+It cannot be installed on existing Operating System, doesn't use RPM repositories and strictly controls most settings using operator pattern.
```

Comment:
FCOS and RHCOS both exist :smile: 
Maybe "It can not be installed on unsupported Operating Systems, ..?"

Reply:
I was trying to say that exising CentOD/RHEL/Ubuntu machines cannot be reused.

How about `It cannot be installed on top of existing OS`?

Reply:
hm `It must be installed on a new installation of a supported OS, the setup on already installed (or provisioned?) machines is not supported?`